components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -967,30 +966,31 @@ public AccessTokenDO getAccessToken(String accessTokenIdentifier, boolean includ
         PreparedStatement prepStmt = null;
         ResultSet resultSet = null;
 
+        if (StringUtils.isEmpty(tenantDomain)) {
+            throw new IdentityOAuth2Exception("Tenant Domain should be needed for further execution");
```

Comment:
Shall we make this message more readable? IMO this message does not clearly give the idea that the tenant domain is not available.

Reply:
Removed with latest commits